strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/MavenGroupRepositoryProviderTest.java

```diff
@@ -45,8 +52,10 @@
 @ContextConfiguration(classes = Maven2LayoutProviderTestConfig.class)
 @Execution(CONCURRENT)
 public class MavenGroupRepositoryProviderTest
-        extends TestCaseWithMavenArtifactGenerationAndIndexing
+        extends MavenMetadataServiceHelper
```

Comment:
@ptirador can you please also inject `MavenMetadataServiceHelper`  instead of using inheritance